### ISSUE TITLE
extract_decoder_hidden_states

### ISSUE THREAD

**Issue by Germany321** [OpenRobotLab/Grounded_3D-LLM#9]
I appreciate the excellent code release. I have a concern about hidden states extraction. Have you ever verified the correctness of the `extract_decoder_hidden_states` function? I tested it locally and found that the predicted logits do not align with the `data_dict.sequences`.
